### DISCUSSION TITLE
[Collaborator/Contributor/Member] titles missing on Issue/PR comments

### DISCUSSION THREAD

**Discussion by Felanbird** [community/community#15908] [Projects and Issues] [answered]
I noticed a few hours ago that the collab/contrib/member titles are missing from user comments, they only properly appear in the body of a PR. 
Bring them back please, I crave validation 🙁

`Issue commenter`
![image](https://user-images.githubusercontent.com/41973452/166229808-eb1e0412-23bb-449a-a4d2-725c7fc1d951.png)

`PR author`
![image](https://user-images.githubusercontent.com/41973452/166229457-a8d0d539-367f-41fc-9f48-15250ee1cd39.png)

**Comment by plainheart** [comment]
It's unexpected for me. Just found it's missing somehow.

**Comment by azenMatt** [comment, marked as answer]
Hi - we recently released a fix for this, let me know if you are still seeing problems

**Reply by Felanbird** [reply]
Seems to be back to normal. 👍
